src/runtime/entitlements-manager-test.js

```diff
@@ -1003,6 +1008,33 @@ describes.realWin('EntitlementsManager', {}, (env) => {
       return manager.consumeMeter_(ents);
     });
 
+    it('should send pingback with metering entitlements and meter params', async () => {
+      const ents = new Entitlements(
+        'service1',
+        'RaW',
+        [
+          new Entitlement(
+            GOOGLE_METERING_SOURCE,
+            ['product1', 'product2'],
+            'token1'
+          ),
+        ],
+        'product1'
+      );
+
+      expectEntitlementPingback(
+        EntitlementSource.GOOGLE_SHOWCASE_METERING_SERVICE,
+        EntitlementResult.UNLOCKED_METER,
+        'token1',
+        GOOGLE_METERING_SOURCE,
+        null,
+        ENTITLEMENTS_URL + '?encodedParams=3ncod3dM3t3ringParams'
+      );
+      expectLog(AnalyticsEvent.EVENT_UNLOCKED_BY_METER, false);
+      manager.encodedParams_ = '3ncod3dM3t3ringParams';
+      return manager.consumeMeter_(ents);
```

Comment:
```suggestion
      await manager.consumeMeter_(ents);
```

If this is equivalent, I think it'd be a bit clearer

Reply:
Done.